content/pages/exec-F15.md

```diff
@@ -0,0 +1,38 @@
+Title: Fall 2015
```

Comment:
Having both a title and h1 means that this gets repeated (see: http://wics-site-pr-168.herokuapp.com/fall-2015/ ). I would set the h1 below to the title up here and omit the h1. (In case unclear, lines prefaced with one hash are formatted as header 1 style, aka h1.)